html/user-activation/consumption-crossorigin.sub.html

```diff
@@ -95,20 +99,27 @@
                 finishReportPhase();
         }
     });
+    async function createIframes() {
+        const child1 = document.createElement("iframe");
+        child1.src = "http://{{hosts[alt][]}}:{{ports[http][0]}}/html/user-activation/resources/child-one.html";
```

Comment:
This change makes "child1" and "grandchild" frames same-origin with each other.  Originally each of the three subframes ("child1", "childXO" and "grandchild") was from a unique origin.

Was this deliberate?  I am asking because in this test, we originally wanted to test 3 different origins in the frame-tree.  If testing two disconnected common-origin frames was the goal, couldn't we add a new grandchild frame to "child1" instead?

FYI: @domenic

Reply:
no, was not deliberate (though given that it's governed at the window level, it shouldn't make a difference - but it's good to test regardless). Let's see if I can find a different origin for "grandchild" that still works with WebKit's infra.

Reply:
Switching the "grandchild" url to `{{hosts[][]}}:{{ports[http][0]}}` removed the same-origin subframes.  I will send out a fix soon.

Reply:
Sent https://github.com/web-platform-tests/wpt/pull/36962 

(checked in WebKit and Chrome)